.pre-commit-config.yaml

```
@@ -0,0 +1,7 @@
+repos:
+  - repo: https://github.com/nbQA-dev/nbQA
+    rev: 0.11.1
```

```suggestion
    rev: 0.12.0
```
Latest version has some small fixes (though nothing that affects this repo). If you use [pre-commit-ci](https://pre-commit.ci/) you'll get a regular (weekly by default, but it's configurable) auto-update

---

Thanks. I don't think I'm able to install GitHub applications into this organization, but I'll keep an eye on it.